### PR TITLE
Add Bug report issue template

### LINKED ISSUE #1008: Create Issue Templates
I just closed more than a dozen stale issues. Quite a few were not reproducible, and having known the users OS, Python and library version involved would've helped a lot. I think that adding templates for issues would've caused users a tiny bit more effort but would help the maintainers a lot.

It would look like this when creating new issues (see the [GitHub docs on these templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)):

new issue page with multiple templates

I'd propose two issue templates: "Bug Report" and "New Feature". We can still leave the option to start with a blank issue. We might consider a simple PR template too, such that no one forgets to close and issue with `Close #007` or `Fixes #007`. Something simple like [this](https://github.com/stevemao/github-issue-templates/tree/master/simple) would probably already suffice.

### PR DESCRIPTION
**Pull request by hardbyte**
Closes #1008

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (felixdivo) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
Steps to reproduce the behavior (please add longer code examples below):
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  
+<details><summary>Traceback and logs</summary>
+<!-- have to be followed by an empty line! -->
+  
+<!-- More details such as a minimal script to demonstrate the bug, relevant logs and any tracebacks go here. -->
+  
+<!-- Code examples can be included: -->
+```python
+  def func():
+      return 'hello, world!'
```
Comment (felixdivo) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]:
```suggestion
def func():
    return 'hello, world!'
```

**Review by felixdivo — APPROVED** [review]
Nice!

**Comment by zariiii9003** [comment]
We could also explicitly ask for OS-version, python version, python-can version and interface

**Comment by felixdivo** [comment]
@Mergifyio rebase

Hope this triggers a merge.

**Comment by hardbyte** [comment]
> @Mergifyio rebase
> 
> Hope this triggers a merge.

Hmm looks like my mergify rules need some work! I'll edit them now following https://docs.mergify.io/examples/#require-all-requested-reviews-to-be-approved

**Pull request merged**